### ISSUE TITLE
DOCKER - Notifications don't work on the Android app with chatwoot

### ISSUE THREAD

**Issue by flavioxx** [chatwoot/chatwoot#9171]
### Describe the bug

When you install Chatwoot in Docker, notifications from the Android app don't work, I tested everything and only when installed directly without Docker do notifications in the Android app work perfectly.

### To Reproduce

.

### Expected behavior

.

### Environment

Docker

### Cloud Provider

Other [please specify in the description]

### Platform

Mobile

### Operating system

.

### Browser and version

.

### Docker (if applicable)

.

### Additional context

.

**Comment by vishnu-narayanan** [comment]
@flavioxx Could you share more about your setup? Are you setting up SSL in both cases?

cc: @muhsin-k

**Comment by flavioxx** [comment]
> @flavioxx Could you share more about your setup? Are you setting up SSL in both cases?
> 
> cc: @muhsin-k

Yes, everything with SSL and working perfectly, but in Docker it doesn't notify in the Android app, whereas in the one installed directly on the server it works.

**Comment by vishnu-narayanan** [comment]
@flavioxx Are you also setting up SSL when using the docker setup?

**Comment by flavioxx** [comment]
> @flavioxx Are you also setting up SSL when using the docker setup?

Yes, I configured SSL as I use it in all Docker applications, I use it together with the nginx-proxy manager in Docker

**Comment by vishnu-narayanan** [comment]
@flavioxx Interesting. So far we haven't recieved any similar reports of notifications not working on docker based setup. Are you seeing any errors on the logs when the application tries to send a push notification?

@muhsin-k Thoughts?

**Comment by flavioxx** [comment]
> @flavioxx Interesting. So far we haven't recieved any similar reports of notifications not working on a docker based setup. Are you seeing any errors on the logs, when the application tries to send a push notification?
> 
> @muhsin-k Thoughts?

It's really strange and I did all the tests and in Docker the Android app doesn't notify me unfortunately, I have to access it to see that the message has arrived.  If you could help me I would be very grateful!

**Comment by shivamkb17** [comment]
Did you enable notification from profile setting? 
This is in setting > Notification preferences

**Comment by a5r0n** [comment]
Same here with kubernetes setup

**Comment by LindezaGrey** [comment]
I noticed that the notifications on Android were no longer working after upgrading from v3.0.0-ce to v3.7.0-ce (Docker)

**Comment by flavioxx** [comment]
> I noticed that the notifications on Android were no longer working after upgrading from v3.0.0-ce to v3.7.0-ce (Docker)

Does the latest version work for you?

**Comment by LindezaGrey** [comment]
i upgraded using the develop-ce tag (updated yesterday). Now the notifications work!

**Comment by flavioxx** [comment]
The version with docker doesn't actually work with push notifications, I wanted to know if it needs any variables to work correctly.
installed directly on the VPS, notifications work perfectly in the Android app.
